src/sentry/static/sentry/images/logos/logo-shortcut.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 80 81" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M80 20.7092C80 9.66108 71.0482 0.709236 60 0.709236H20C14.8847 0.709236 9.76123 2.65986 5.85998 6.56921C-1.95063 14.3798 -1.95063 27.0387 5.85998 34.8493L12.9826 41.9719C5.39862 44.8209 0 52.1378 0 60.7092C0 71.7574 8.95184 80.7092 20 80.7092H60C65.1153 80.7092 70.2388 78.7586 74.14 74.8493C81.9506 67.0387 81.9506 54.3798 74.14 46.5692L67.0174 39.4466C74.6014 36.6056 80 29.2888 80 20.7092ZM11.3476 69.3697C9.03278 67.0548 7.76204 63.9873 7.76204 60.7173C7.76204 57.4474 9.03278 54.3717 11.3476 52.065C13.5411 49.8715 16.4144 48.617 19.4901 48.4875L43.9579 72.9553H20C16.7301 72.9472 13.6625 71.6765 11.3476 69.3697ZM68.6524 52.0569C70.9672 54.3717 72.238 57.4393 72.238 60.7092C72.238 63.9792 70.9672 67.0548 68.6524 69.3616C66.3375 71.6765 63.2699 72.9472 60 72.9472C56.7301 72.9472 53.6544 71.6765 51.3476 69.3616L11.3476 29.3697C9.03278 27.0548 7.76204 23.9873 7.76204 20.7173C7.76204 17.4474 9.03278 14.3717 11.3476 12.065C13.6625 9.75011 16.7301 8.47937 20 8.47937C23.2699 8.47937 26.3456 9.75011 28.6524 12.065L68.6524 52.0569ZM68.6524 29.3697C66.4589 31.5631 63.5856 32.8177 60.5099 32.9472L36.0421 8.47128H60C63.2699 8.47128 66.3456 9.74202 68.6524 12.0569C70.9672 14.3717 72.238 17.4393 72.238 20.7092C72.238 23.9792 70.9672 27.0548 68.6524 29.3697Z" fill="#58B1E4"/>
+</svg>
```

Comment:
@robinrendle  I'm not sure how SVGs work but @ceorourke's point was valid.  I just faked some react props to test it, do you know how to change the SVG to work like the old one?

![image](https://user-images.githubusercontent.com/35509934/133854581-9338f414-073a-41d4-ac9e-c36e29fead9b.png)

Reply:
Okay – fixed. I figured out the problem with the SVG I was using.